pubspec.yaml

```diff
@@ -33,9 +34,9 @@ dependencies:
   cupertino_icons: ^1.0.2
   flutter_platform_widgets: ^1.9.5
   font_awesome_flutter: ^9.1.0
-  flutter_riverpod: ^1.0.0-dev.7
-  image_picker: ^0.8.4+4
-  video_player: ^2.2.7
+  flutter_riverpod: ^1.0.0
+  http: ^0.13.4
+  provider: ^6.0.1
```

Comment:
You should be using riverpod not provider! If you are not using please remove this dependency.